docs/user/software/development/java.md

```diff
@@ -1,34 +1,40 @@
 ---
 title: Java
-summary: A quick guide to Java development on Solus
+summary: Learn how to install, run, and manage Java on Solus.
 ---
 
 # Java
 
-This article covers obtaining open source Java (otherwise known as OpenJDK). In this article, two abbreviations will be used to refer to components of the Java software suite:
+Solus includes multiple LTS versions of Java in the repositories. You can install more than one version in your system at once.
 
-- JRE (Java Runtime Environment)
-- JDK (Java Development Kit)
+:::tip
 
-## Installing OpenJDK
+If you need to install different versions or Java, use alternative installation methods such as _SDKMAN!_.
 
-Solus currently offers two Java versions in the repository, and these versions can be installed alongside one another. For ease of maintenance, Solus only includes LTS versions of OpenJDK. Those who require the absolute newest version of the JDK, or other non-LTS versions, should look to other methods of installation (such as [sdkman](https://sdkman.io/)).
+:::
 
-### OpenJDK 11
+All Java packages in the Solus repositories include the _Java Runtime Environment_ (JRE) and the _Java Development Kit_ (JDK).
 
-OpenJDK 11 is included in the repository as `openjdk-11`. It includes the JRE, the JDK, and OpenJFX, all in one package. The majority of Java packages in Solus currently use OpenJDK 11, and this is the recommended version if you need Java but aren't sure which version you should install. If the software you use doesn't work with OpenJDK 11, you should use OpenJDK 8 instead.
+The following table lists the versions of Java available in the Solus repositories.
```

Comment:
```suggestion
The following table lists the versions of Java available in the Solus repositories:
```

Reply:
Since this is 100% a style choice, which style should the documents follow moving forward? For reference, [Google](https://developers.google.com/style/tables) is pretty flexible with it, whereas [Microsoft](https://learn.microsoft.com/en-us/style-guide/scannable-content/tables#punctuation) prefers using periods only.